website/integrations/services/openproject/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: OpenProject
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is OpenProject
+
+> OpenProject is a open source project management software.
+>
+> -- https://www.openproject.org/
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `openproject.company` is the FQDN of the Service install. (Remove this for SaaS)
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik Configuration
+
+Create a [OAuth2/OpenID provider](https://goauthentik.io/docs/providers/oauth2) with the following parameters:
+
+-   Client Type: `Confidential`
+-   Scopes: `openid`, `email` and `profile`
+-   Signing Key: Select any available key
+-   Redirect URIs: `https://openproject.company/auth/authentik/callback`
+
+Note the `Client ID` and `Client Secret` values.
+Create an [application](https://goauthentik.io/docs/applications), using the provider you've created above and set a slug.
```

Comment:
```suggestion
Create an application, using the provider you've created above and set a slug.
```

Reply:
The reason we don't want to link to the Applications docs (right now) is because the docs do not yet have steps for how to *create* an Application, and that is what this line 30 is about. We are about to update our Applications docs to include procedurals for using the new Wizard to create apps/providers. 

So let's remove this link, please.

Reply:
Ooops, and now we do have instructions! ;-) So let's leave the link, but edit it to point to https://docs.goauthentik.io/docs/applications/manage_apps.

@benszau I hope it is ok with you if we make the changes and go ahead and merge. Or let us know if you want to do so, or add more content, etc.